source/about/security.rst

```diff
@@ -35,6 +35,12 @@ Transmission security
 ~~~~~~~~~~~~~~~~~~~~~~
 
 - Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server across both LAN and internet.
+- Connections to calls are secured with a combination of:
```

Comment:
0/5 is it clear enough that this is an exception to the above?

Reply:
@streamer45 ^